_posts/2020-04-20-origami-survey-results.md

```diff
@@ -0,0 +1,204 @@
+---
+title: Origami Survey Results, Building Components
+description: TODO
+author: Rowan Manning
+hasCharts: true
+---
+
+{% assign results = site.data.survey['2020-04-20'] %}
+{% assign areas = "CRM, Customer Products, Enterprise Services and Security, FT Core, FT Group Products, Internal Products, Operations and Reliability, Product" | split: ", " %}
+
+
+We recently sent out an internal survey titled "Origami Survey: Building Components". The survey was designed to gather some quantititive data from our users, and the aim was to help us understand the barriers that teams have to developing their own Origami (or Origami-compatible) components. We sent the survey to engineers, and indicated that we were interested in responses from engineers of all seniorities and levels of experience with Origami.
+
+After two weeks, we had **{{results | size}} responses** from different groups within Product and Technology. This post will explore some of the results, and what we can learn from them.
+
+To help segment the data, we asked each respondent which area of Product and Technology they worked in, as well as asking them to self-report their level of familiarity with using Origami components.
+
+We'd like to say a big thanks to everyone who filled out the survey 🙂
```

Comment:
Should we add a full stop after the emoji here?
```suggestion
We'd like to say a big thanks to everyone who filled out the survey 🙂.
```

Reply:
I'm never sure with emoji, I don't know if we have an FT style-guide for this :D I might ask somewhere on Slack what people think

Reply:
Based on a chat in #new-emoji-elephant, there is no FT style, and I think I prefer no punctuation